rikaicontent.js

```diff
@@ -788,29 +809,46 @@ var rcxContent = {
 	},
 	
 	getTotalOffset: function(parent, tNode, offset) {
-	    var fChild = parent.firstChild;
-	    var realO = offset;
-	    if(fChild == tNode)
-	        return offset;
-	    do {
-	        var val = 0;
-	        if(fChild.nodeName == "BR") {
-	            val = 1;
-	        }
-	        else {
-	            val = (fChild.data ? fChild.data.length : 0)
-	        }
-	        realO += val;
-	    }
-	    while((fChild = fChild.nextSibling) != tNode);
-	    
-	    return realO;
+		var fChild = parent.firstChild;
+		var realO = offset;
+		if(fChild == tNode)
+			return offset;
+		do {
+			var val = 0;
+			if(fChild.nodeName == "BR") {
+				val = 1;
+			}
+			else {
+				val = (fChild.data ? fChild.data.length : 0)
+			}
+			realO += val;
+		}
+		while((fChild = fChild.nextSibling) != tNode);
+		
+		return realO;
 	
 	},
 	
-	onMouseMove: function(ev) { rcxContent._onMouseMove(ev); },
-	_onMouseMove: function(ev) {
+	onMouseMove: function(ev) { 
+		rcxContent.lastPos.x = ev.clientX;
+		rcxContent.lastPos.y = ev.clientY;
+		rcxContent.lastTarget = ev.target;
+		rcxContent.tryUpdatePopup(ev); 
+	},
+	tryUpdatePopup: function(ev) {
+
+		if ((window.rikaichan.config.showOnKey.includes("Alt") && !ev.altKey) ||
+			 (window.rikaichan.config.showOnKey.includes("Ctrl") && !ev.ctrlKey)) {
+			this.clearHi();
+			this.hidePopup();
+			return;
+		}
+
 		var fake;
+		var tdata = window.rikaichan;	// per-tab data

```

Comment:
Because JS is amazing of course, anything is possible.  (github diffs are hard for me to read so at first I actually didn't see it was outside the try)

This code could really benefit from a re-write since I barely refactored at all when I ported rikaichan.